bindings/guile/gnc-guile-bindings.c

```diff
@@ -44,6 +44,5 @@ gnc_guile_bindings_init(void)
         scm_init_sw_core_utils_module();
         scm_init_sw_engine_module();
 
-        is_initialized = 1;
```

Comment:
This seems inconsistent:
* either we use is_initialized to do initialization only once. In that case is should be set here.
* or we don't test for is_initialized at all and potentially take the overhead of initializing more than once if the init function is called more than once.

Why was it removed ?

Reply:
Note I traced this back to commit 90b8fce59fe02a0e0ccac05a2baaadf7317e07a0
It's not clear from that commit's message what exactly the problem was with this.

Reply:
I don't remember. It might have gotten in the way of swigging app-utils early on; that commit was from a year ago. I just checked and it doesn't matter now. I've put it back.